var/spack/repos/builtin/packages/py-torchtext/package.py

```diff
@@ -11,17 +11,53 @@ class PyTorchtext(PythonPackage):
     """Text utilities and datasets for PyTorch."""
 
     homepage = "https://github.com/pytorch/text"
-    pypi = "torchtext/torchtext-0.5.0.tar.gz"
+    git = "https://github.com/pytorch/text.git"
 
     maintainers = ["adamjstewart"]
 
-    version("0.5.0", sha256="7f22e24e9b939fff56b9118c78dc07aafec8dcc67164de15b9b5ed339e4179c6")
-
-    depends_on("python@2.7:2.8,3.5:", type=("build", "run"))
+    version("main", branch="main", submodules=True)
+    # https://github.com/pytorch/text/issues/1976
+    version("0.14.0", tag="v1.14.0", submodules=True)
```

Comment:
Would like to hold off on merging this until this issue is resolved. This will likely result in a changed tag name.